src/qt/bitcoingui.cpp

```diff
@@ -368,12 +369,17 @@ void BitcoinGUI::createActions()
     m_mask_values_action->setStatusTip(tr("Mask the values in the Overview tab"));
     m_mask_values_action->setCheckable(true);
 
+    getRawTransactionAction = new QAction(tr("&Verify external txid"), this);
```

Comment:
nit:
```suggestion
    getRawTransactionAction = new QAction(tr("&Verify transaction"), this);
```

Some thoughts: perhaps we use a more generic naming here and in the form we indicate if it's a wallet transaction with a checkbox (and then we change the form removing the blockhash option and call `gettransaction`?). Or maybe just on a follow-up.